src/components/SignUp/CareerInfo.js

```diff
@@ -74,6 +75,20 @@ class CareerInfo extends Component {
       Actions.refresh({ rightTitle: 'SAVE', onRight: this.onNextBtnPressed.bind(this) });
   }
 
+  componentWillReceiveProps(props) {
+    if(props.fromEdit && this.state.needRefresh) {
```

Comment:
@ochanje210 What is prop validation? I never heard of that. could you give me a brief explanation or related link? and if it doesn't have fromEdit property, it will evaluated as false, thus, it will not pass that if condition. Therefore, that if condition will executed only when this component has fromEdit props.